src/commands/create.ts

```diff
@@ -134,6 +134,7 @@ export const createQueryDisputeCommand = {
         chainId,
         contracts.disputeManager.address,
         attestation,
+        '0',
```

Comment:
Should this be a command option?

Reply:
We have a single query version for now, but totally consider for when we add new ones.